### DISCUSSION TITLE
Possible to get information about the driver?

### DISCUSSION THREAD

**Discussion by DcFamas** [bassmaster187/TeslaLogger#1065] [General]
Hey there,
is it possible to get the information, which person is driving?
Background is, that I want to create a logbook to split the cost of the vehicle.

Best regards :)

**Comment by adnet-solutions** [comment]
as i know the data is not available or included in the API. i had the same request some time ago.
Also the information which profile is loaded or which phone is connected is not available in Tesla API. And without that teslalogger cant read the data.
maybe a way is to get it from scan my tesla as that one have maybe more informations but thats a question to @bassmaster187 if he see any way to get that information available in teslalogger. For me its would also helpfull.

**Comment by AISCPO** [comment]
There is right now not possible to get the info of the driving person. I asked tesla a couple of times about the current profile, but right now there is no plan to include it in their data stream.

**Comment by bassmaster187** [comment]
Maybe some upvotes here to get this feature:
https://github.com/teslamotors/fleet-telemetry/issues/352#issuecomment-4062635063

**Comment by adnet-solutions** [comment]
@bassmaster187 wie es aussieht kann mit dem neuem Teslupdate ja die Teslaapp zumindest den aktiven Fahrer erkennen. Gibt es ne chance das sie das in die api eingebunden haben?
